static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.spec.js

```diff
@@ -150,6 +171,46 @@ describe('init', () => {
         expect(prebid.initialise).not.toBeCalled();
     });
 
+    it('should initialise Prebid in CCPA if doNotSell is false', async () => {
+        dfpEnv.hbImpl = { prebid: true, a9: false };
+        commercialFeatures.dfpAdvertising = true;
+        commercialFeatures.adFree = false;
+        onConsentChange.mockImplementation(ccpaWithConsentMock);
+        getConsentFor.mockReturnValue(true); // TODO: Why do we need to mock this?
```

Comment:
I wish we didn’t mock this…

Reply:
last time to use the real one it was giving ES2020 errors like this one: https://chat.google.com/room/AAAAWwBdSMs/kxMrGKQfdiY